### PR TITLE
Update Microsoft defender endpoint readme 

### PR DESCRIPTION
**Pull request by cherryleaf**
Type of change

- Enhancement

## What does this PR do?

Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing 

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

File: packages/microsoft_defender_endpoint/_dev/build/docs/README.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
-This integration is for Microsoft Defender for Endpoint logs.
+This integration is for [Microsoft Defender for Endpoint](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint?view=o365-worldwide) logs.
+
+## Setting up
 
 To allow the integration to ingest data from the Microsoft Defender API, you need to create a new application on your Azure domain. The procedure to create an application is found on the [Create a new Azure Application](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/exposed-apis-create-app-webapp) documentation page.
 
-When giving the application the API permissions described in the documentation (`Windows Defender ATP Alert.Read.All`) it will only grant access to read alerts from ATP and nothing else in the Azure Domain.
+> Note: When giving the application the API permissions described in the documentation (`Windows Defender ATP Alert.Read.All`), it will only grant access to read alerts from ATP and nothing else in the Azure Domain.
```
Comment (bmorelli25) [inline packages/microsoft_defender_endpoint/_dev/build/docs/README.md:9]: Is the `>` intentional here?

Comment (GinnyGalicia) [reply packages/microsoft_defender_endpoint/_dev/build/docs/README.md:9]: Yes, it generates a Markdown admonition note box

Comment (GinnyGalicia) [reply packages/microsoft_defender_endpoint/_dev/build/docs/README.md]: I've changed this to Elastic's DocCallOut tag

**Review by andrewkroh — APPROVED** [review]

File: packages/microsoft_defender_endpoint/_dev/build/docs/README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-This integration is for Microsoft Defender for Endpoint logs.
+This integration is for [Microsoft Defender for Endpoint](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/microsoft-defender-endpoint?view=o365-worldwide) logs.
+
+## Setting up
 
 To allow the integration to ingest data from the Microsoft Defender API, you need to create a new application on your Azure domain. The procedure to create an application is found on the [Create a new Azure Application](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/exposed-apis-create-app-webapp) documentation page.
 
-When giving the application the API permissions described in the documentation (`Windows Defender ATP Alert.Read.All`) it will only grant access to read alerts from ATP and nothing else in the Azure Domain.
+<DocCallOut title="Note">
+When giving the application the API permissions described in the documentation (`Windows Defender ATP Alert.Read.All`), it will only grant access to read alerts from ATP and nothing else in the Azure Domain.</DocCallOut>
```
Comment (bmorelli25) [inline packages/microsoft_defender_endpoint/_dev/build/docs/README.md:10]: This needs to be reverted as it won't render in markdown.

**Review by bmorelli25 — CHANGES_REQUESTED** [review]

File: packages/microsoft_defender_endpoint/_dev/build/docs/README.md (hunk 2 above)
Comment (ellis-elastic) [reply packages/microsoft_defender_endpoint/_dev/build/docs/README.md:10]: I'm going to have to raise a new PR, as I cant' seem to edit this.

**Comment by ellis-elastic** [comment]
I'm going to have to raise a new PR, as I cant' seem to edit this.

**Pull request closed without merging**